### PR TITLE
Publish docker image from a tag

### PR DESCRIPTION
**Pull request by tanandy**
This will allow developers to automatically create docker image based on a git tag

### CODE REVIEW

**Review by oubchid — DISMISSED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -9,44 +9,56 @@ pipeline {
...
           args '-e DOCKER_HOST=$DOCKER_HOST'
         }
       }
-
       steps {
         script {
           def dockerImage = docker.build 'linagora/esn-frontend-inbox'
           docker.withRegistry('', 'dockerHub') {
-            dockerImage.push('branch-main')
+            dockerImage.push('latest')
```
Comment (fabienmoyon) [inline Jenkinsfile:35]: you will change all others Jenkinsfile to `latest` ?

**Comment by chibenwa** [comment]
Could we be lucky enough to get a more descriptive commit message?

**Review by Arsnael — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (tanandy) [reply Jenkinsfile:35]: yes exactly

**Review by tanandy — COMMENTED** [review]
commented

**Review by fabienmoyon — APPROVED** [review]

**Review by oubchid — APPROVED** [review]

**Review by alagane — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Pull request merged**